tests/README.md

```diff
@@ -51,6 +51,19 @@ bash tests/docker/ducker-ak up -j 'openjdk:11'; tests/docker/run_tests.sh
 ```
 REBUILD="t" bash tests/docker/run_tests.sh
 ```
+* Run tests with Kafka in `native` mode
```

Comment:
Will the system tests report generated using native mode will be included in RC voting? Do we have any process where this will be ran as part of release or on a regular basis? It's fine if it's currently not addressed but will be addressed in future.

Reply:
I have created a job internally and I would share the system test results(for the Apacha Kafka Native executable) during RC voting.
Currently I have not decided the frequency of the run but it will definitely be run as the part of the release.